src/test/java/com/hotmail/kalebmarc/textfighter/main/VersionTest.java

```diff
@@ -0,0 +1,35 @@
+package com.hotmail.kalebmarc.textfighter.main;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class VersionTest {
+
+    @Test
+    public void get() {
+        assertEquals(Version.get(), "4.9DEV");
+    }
+
+    @Test
+    public void getStage() {
+        assertTrue(Version.getStage() == "Alpha");
```

Comment:
When comparing strings, its better to use `.equals`
```suggestion
        assertTrue(Version.getStage().equals("Alpha"));
```